variables.tf

```diff
@@ -3,31 +3,17 @@ variable "bucket_name" {
   description = "S3 bucket where salt repo will be mirrored"
 }
 
-variable "salt_versions" {
-  type        = list(string)
-  description = "List of salt versions to mirror; will also generate version-specific yum .repo definition files"
-  default     = []
-}
-
 variable "repo_endpoint" {
   type        = string
   description = "HTTP/S endpoint URL that hosts the yum repos; used with the baseurl in the yum .repo definitions"
 }
 
-variable "repo_prefix" {
-  type        = string
-  description = "S3 key prefix where the repos will be mirrored"
-  default     = ""
-}
-
-variable "yum_prefix" {
-  type        = string
-  description = "S3 key where the yum repo definitions will be placed"
-  default     = ""
-}
-
-variable "salt_s3_endpoint" {
-  type        = string
-  description = "S3 endpoint for the upstream salt repo"
-  default     = "https://s3.repo.saltstack.com"
+variable "repos" {
```

Comment:
Maybe add a description summarizing what each key is for?  The generated README doesn't have any information whereas before there was a nice explanation of what each variable was for.

Reply:
Terraform objects can't really store a description per attribute, so that is something we lose when we use objects like this. I gave it a shot in the variable-level description, but this description is something has to be updated manually and it can get unwieldy as the object grows. Where we've tried this elsewhere, it usually ends up becoming out-dated... Anyway, it's there now.

Reply:
Update looks good.  I see your point as objects grows.